src/features/query-bar/__tests__/Condition-test.js

```diff
@@ -1,9 +1,11 @@
 // @flow
 import * as React from 'react';
 
-import { columnWithEnumType, initialCondition } from '../components/fixtures';
+import { columnWithEnumType } from '../components/fixtures';
 import Condition from '../components/filter/Condition';
+import { EQUALS } from '../constants';
 
+const invalidCondition: ConditionType = { columnId: '3', id: '0', operator: EQUALS, values: [] };
```

Comment:
'3' should be replaced with columnForTemplateFieldName.id (or whatever column uses id=3).